t/mojo/promise_async_await.t

```diff
@@ -157,6 +162,15 @@ subtest 'Async function body returning a promise' => sub {
   is $text, 'value', 'right content';
 };
 
+subtest 'Async nested function returning a promise' => sub {
+  my $res;
+  await_then_return_promise('timer')->then(sub { $res = shift })->catch(sub { $res = shift })->wait;
+  is $res, 'timer', 'right content';
+
+  await_then_return_promise('timeout')->then(sub { $res = shift })->catch(sub { $res = shift })->wait;
+  like $res, qr{^timeout at}, 'right content';
```

Comment:
Why not the standard `qr/.../`?

Reply:
No good reason. I personally have changed to `qr{...}` since I more often have "/" inside the regexp than "{", so it's just a matter of habit.